easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
+
+name = 'pytest'
+version = '3.8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pytest.org'
+description = """pytest: simple powerful testing with Python"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [('Python', '3.6.4')]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
```

Comment:
with PythonBundle, `exts_defaultclass` and `exts_default_options` can be removed. instead use just this:
```
use_pip = True
```